src/components/DarkMode/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styles from "./DarkMode.module.css";
+
+const DarkMode: React.FC = () => {
+  let storedTheme = "light"
+  let prefersDark = false
+  const [theme, setTheme] = React.useState<string>('light')
+
+  const setDark = () => {
+    localStorage.setItem("theme", "dark");
+    document.documentElement.setAttribute("data-theme", "dark");
+  }
+  
+  const setLight = () => {
+    localStorage.setItem("theme", "light");
+    document.documentElement.setAttribute("data-theme", "light");
+  }
+  
+  React.useEffect(() => {
+    storedTheme = localStorage.getItem("theme")!;
+  }, [])
```

Comment:
Don't use variables inside a component, as their value will reset on every render. I don't think `storedTheme`  is required here, you can store the value in `theme` itself.